synapse/cortex.py

```diff
@@ -3168,6 +3168,122 @@ async def watch(self, wdef):
             async for mesg in wind:
                 yield mesg
 
+    async def getExtModel(self):
+        '''
+        Get all extended model properties in the Cortex.
+
+        Returns:
+            dict: A dictionary containing forms, form properties, universal properties and tag properties.
+        '''
+        ret = collections.defaultdict(list)
+        for formname, basetype, typeopts, typeinfo in self.extforms.values():
+            ret['forms'].append({'formname': formname,
```

Comment:
Perhaps these should be in the same format as they are specified to the individual CRUD API. ( quick goback )